server/src/test/java/org/candlepin/policy/AutobindRulesTest.java

```diff
@@ -2707,14 +2711,14 @@ public void testShouldSelectPoolWhenRoleMatchesButOtherSysPurposeAttributesAndSo
         prodMCT1650.setAttribute(Product.Attributes.RAM, "19960912");
         Pool MCT1650 = TestUtil.createPool(owner, prodMCT1650);
         MCT1650.setId("MCT1650");
-        MCT1650.addProvidedProduct(product69);
+        MCT1650.getProduct().addProvidedProduct(product69);
```

Comment:
Add these to the product directly during product declaration.

Reply:
Updated.